discovery-provider/es-indexer/src/indexers/PlaylistIndexer.ts

```diff
@@ -95,6 +95,7 @@ export class PlaylistIndexer extends BaseIndexer<PlaylistDoc> {
             save_count: { type: 'integer' },
```

Comment:
You will also need to force elastic-search to reindex playlists. This can be done by modifying this
https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/es-indexer/src/indexNames.ts#L2
to playlists14, which on next start up of ES will trigger index creation blue/green style